modules/utxo-bin/README.md

```diff
@@ -9,6 +9,14 @@ formatting them for terminal output.
 npm install --global @bitgo/utxo-bin
 ```
 
+# Upgrade
+```
+cd modules/utxo-bin
+yarn build
+```
+
+Either run using ``yarn ts-node bin/index.ts ...`` or install globally again using previous npm command.
```

Comment:
the `yarn ts-node bin/index.ts` should be mentioned in a section `Usage` / `From Repo`

we should probably add a new section `Installation / From Repo` with `cd modules/utxo-bin; npm install --global`